HyperPlay-Gaming/hyperplay-desktop-client
# [TECH] Logs Cleanup

- Changed the `stopChildProcesses` method to only log on Abort and for games only since it was logging all commands before.
- Removed most of other useless logs like the ones that says your are logged in or not on GOG and EPIC.
- Removed others that were added when some features were new and we were debugging like from abort controller and more.
- Didn't remove any HyperPlay logs.

### How to test

This should be easy to test and should not break anything since it is just logs removal. But since there were some changes to the `stopChildProcesses` method, would be good to test it again to see if games like XOciety will still close completely.

To test the removed logs do some actions like refresh the libraries, launch a game, exit, launch again, abort it from the overlay or the STOP button and check the HyperPlay logs (not game logs since they are not affected).

Logs including `PID` and `abortController` should not appear when refreshing library or closing the game normally. But the `Killing child process` ones should appear when the game is aborted from the client.


---

Use the following Checklist if you have changed something on the Backend or Frontend:
